cli_tools/diagnostics/rdp_status.ps1

```diff
@@ -33,4 +49,4 @@ foreach($item in $items){
   Write-Host $item["description"]
   Invoke-Expression $item["cmd"]
   Write-Host
-}
\ No newline at end of file
+}
```

Comment:
Identifying if the instance has the Remote Desktop feature enabled (ie it can support more than 2 users) would be very helpful as when that is enable the customer needs to set the "connect to an admin session" for adminstrative operations.

Because it changes the connection setting requirements it's an incredibly important detail to collect. We've had support cases where RDP licensing wasn't configured or out of the grace period and in that state only admin sessions are allowed and the resolution was setting the customer to set that setting in their RDP client.

Reply:
Added

Reply:
I'm not sure that "(Get-Service -Name TSGateway).Status" will accomplish this goal as that's specifically around the Gateway capability.

https://learn.microsoft.com/en-us/windows/win32/termserv/detecting-whether-terminal-services-is-installed

get-windowsfeature RDS-RD-Server
```
Display Name                                            Name                       Install State
------------                                            ----                       -------------
    [ ] Remote Desktop Session Host                     RDS-RD-Server                  Available
```

Reply:
Please I have updated this, thank you so much for your support